src/pages/settings/InitialSettingsPage.js

```diff
@@ -138,7 +143,7 @@ class InitialSettingsPage extends React.Component {
                 translationKey: 'common.payments',
                 icon: Expensicons.Wallet,
                 action: () => { Navigation.navigate(ROUTES.SETTINGS_PAYMENTS); },
-                brickRoadIndicator: PaymentMethods.hasPaymentMethodError(this.props.bankAccountList, this.props.cardList) || !_.isEmpty(this.props.userWallet.errors) ? 'error' : null,
+                brickRoadIndicator: PaymentMethods.hasPaymentMethodError(this.props.bankAccountList, this.props.cardList) || !_.isEmpty(this.props.userWallet.errors) || !_.isEmpty(this.props.walletTerms.errors) ? 'error' : null,
```

Comment:
The linter is complaining about this line being too long. Guess it's time to split it into multiple lines :)